campaign/record_power.xml

```diff
@@ -109,15 +118,22 @@
 				fOnDataChanged = super.onDataChanged
 				super.onDataChanged = onDataChanged
 				local sNode = getDatabaseNode().getPath();
+
+				super.onInit();
```

Comment:
Apparently I can't leave comments on unchanged code... Do you think this has grown large enough to justify it's own script file?

Reply:
I moved most of the scripts from power classes to their own lua files.